guides/source/debugging_rails_applications.md

```diff
@@ -107,16 +107,15 @@ It can also be useful to save information to log files at runtime. Rails maintai
 
 ### What is the Logger?
 
-Rails makes use of the `ActiveSupport::Logger` class to write log information. Other loggers, such as `Log4r`, may also be substituted.
-
-You can specify an alternative logger in `config/application.rb` or any other environment file, for example:
+Rails makes use of the `ActiveSupport::Logger` class to write log information. Other loggers, such as `Log4r`, may be substituted:
 
 ```ruby
+# config/application.rb
 config.logger = Logger.new(STDOUT)
 config.logger = Log4r::Logger.new("Application Log")
 ```
 
-Or in the `Initializer` section, add _any_ of the following
+Or in the `Initializer` section, add _any_ of the following:
```

Comment:
I'd probably remove this line and the following example.
I don't think we should be advising configuring logger this way as the environment config seems the most common location.

Reply:
removed as suggested.